api/src/models/Config.ts

```diff
@@ -224,6 +224,14 @@ class Config {
     get licenses(): Record<string, License> {
         return this.ini["licenses"] || {};
     }
+
+    get agentRoles(): Array<string> {
+        return this.ini["agent"]["roles"]
```

Comment:
Might be good to have these default to something (even if just an empty array) if the configuration is undefined -- I forgot to update vudl.ini at first and everything broke.